Examples/Scripts/PostFreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,243 @@
+	job_list+=("$job")
+	#DEBUG
+	#break
+  done
+  jl="${job_list[@]}"
+
+  echo "Launched stage 1 timepoint jobs: $jl (waiting for the prep template job $template_job)"
+
+  #process template. Must finish before timepoints are processed if MSMSulc is run.
+  echo "template processing"
+  template_job=$($queuing_command -j ${jl// /,} "$HCPPIPEDIR"/PostFreeSurfer/PostFreeSurferPipeline.sh \
+      --study-folder="$StudyFolder" \
+      --subject="$Subject" \
+      --surfatlasdir="$SurfaceAtlasDIR" \
+      --grayordinatesdir="$GrayordinatesSpaceDIR" \
+      --grayordinatesres="$GrayordinatesResolution" \
+      --hiresmesh="$HighResMesh" \
+      --lowresmesh="$LowResMeshes" \
+      --subcortgraylabels="$SubcorticalGrayLabels" \
+      --freesurferlabels="$FreeSurferLabels" \
+      --refmyelinmaps="$ReferenceMyelinMaps" \
+      --regname="$RegName" \
+      --longitudinal_mode="TEMPLATE" \
+      --longitudinal_template="$LongitudinalTemplate" \
+      --longitudinal_timepoint_list="${Timepoint_list[i]}" \
+      --longitudinal_timepoint="$Timepoint")
+
+  echo "Launched template job $template_job"
+  echo "Template job $template_job will wait for Stage 1 timepoint jobs: $jl" 
+  #DEBUG
+  #break;
+
+  job_list=()
+  for Timepoint in ${Timepoints[@]}; do
+  	#input variables specific for timepoint mode (if any)
+  	# ...
+	
+	#DEBUG
+	#continue
+	#process each timepoint
+	job=$($queuing_command -j $template_job "$HCPPIPEDIR"/PostFreeSurfer/PostFreeSurferPipeline.sh \
+	      --study-folder="$StudyFolder" \
+	      --subject="$Subject" \
+	      --surfatlasdir="$SurfaceAtlasDIR" \
+	      --grayordinatesdir="$GrayordinatesSpaceDIR" \
+	      --grayordinatesres="$GrayordinatesResolution" \
+	      --hiresmesh="$HighResMesh" \
+	      --lowresmesh="$LowResMeshes" \
+	      --subcortgraylabels="$SubcorticalGrayLabels" \
+	      --freesurferlabels="$FreeSurferLabels" \
+	      --refmyelinmaps="$ReferenceMyelinMaps" \
+	      --regname="$RegName" \
+	      --longitudinal_mode="TIMEPOINT_STAGE2" \
+	      --longitudinal_template="$LongitudinalTemplate" \
+	      --longitudinal_timepoint="$Timepoint")
+    job_list+=("$job")
+	#DEBUG
+	#break
+  done
+  echo "launched stage 2 timepoint jobs: ${job_list[@]}"
+  echo "Stage 2 timepoint jobs (${job_list[*]}) will wait for the template job: $template_job"
+    
+  # The following lines are used for interactive debugging to set the positional parameters: $1 $2 $3 ...
+  
+  # echo "set -- --study-folder=$StudyFolder \
+  #    --subject=$Subject \
+  #    --surfatlasdir=$SurfaceAtlasDIR \
+  #    --grayordinatesdir=$GrayordinatesSpaceDIR \
+  #    --grayordinatesres=$GrayordinatesResolutions \
+  #    --hiresmesh=$HighResMesh \
+  #    --lowresmesh=$LowResMeshes \
+  #    --subcortgraylabels=$SubcorticalGrayLabels \
+  #    --freesurferlabels=$FreeSurferLabels \
+  #    --refmyelinmaps=$ReferenceMyelinMaps \
+  #    --regname=$RegName"
+      
+  # echo ". ${EnvironmentScript}"
+done
+
```

Comment:
Does all this stuff work if fsl_sub is not available?  We cannot count on it on, for example, the CHPC.

Reply:
Also, I don't think we should have the Batch doing the Pipeline chain of operations.  That should be in a script called by the Batch.

Reply:
1. fsl_sub is available on CHPC, althogh it should be tested.
2. Pipeline logic is moved to PostFreeSurferPipelineLongLauncher.sh, with necessary changes in the PostFSPipeline batch script.